docs/source/1.0/spec/core/behavior-traits.rst

```diff
@@ -655,3 +655,134 @@ See
             input: PutSomethingInput,
             output: PutSomethingOutput
         }
+
+
+.. _httpChecksum-trait:
+
+----------------------
+``httpChecksum`` trait
+----------------------
+
+Summary
+    Indicates that an operation's HTTP Request or Response supports checksum
+    validation. At least one of request or response checksum properties must
+    be specified within the trait.
+
+    When this trait is modeled along with
```

Comment:
This trait needs a section to define behavior for it [like `@paginated`](https://awslabs.github.io/smithy/1.0/spec/core/behavior-traits.html#paginated-trait), instead of sprinkling it in to the trait documentation.

Reply:
Defined a behavior section.